src/einsteinpy/plotting/geodesics_scatter.py

```diff
@@ -40,6 +41,53 @@ def plot(self, pos_vec, vel_vec, end_lambda=10, step_size=1e-3):
         if not self._attractor_present:
             self._plot_attractor()
 
+    def plot_animated(
+        self, pos_vec, vel_vec, end_lambda=10, step_size=1e-3, interval=50
+    ):
+        """
+
+        Parameters
+        ----------
+        Interval: Control the time between frames. Add time in milliseconds.
+
+        Other parameters are same as plot method.
```

Comment:
Please mention everything. Moreover, I see `interval` in the function declaration. Also, The proper way of writing docstring would be: 

```
Parameters
----------------
interval : int, optional
    Control the time between frames. Add time in milliseconds.
```